exporter/prometheusremotewriteexporter/config_test.go

```diff
@@ -136,3 +141,8 @@ func TestDisabledTargetInfo(t *testing.T) {
 
 	assert.False(t, cfg.(*Config).TargetInfo.Enabled)
 }
+
+func toPtr[T any](val T) *T {
+	ret := val
+	return &ret
+}
```

Comment:
Go is pass by copy, so it is fine to take the address of the val

Reply:
ur right. updated.